source/release-notes/4-2-x-breaking-changes.rst

```diff
@@ -0,0 +1,32 @@
+:orphan:
+:is-up-to-date: False
+:last-updated: 4.2.0
+
+.. _breaking-changes-4-2-x:
+
+====================================
+Breaking Changes in CrafterCMS 4.2.x
+====================================
+This section covers changes that might affect your CrafterCMS projects, as well as other considerations
+before upgrading. Please review the following and apply changes as required:
+
+- CrafterCMS 4.2.x requires Java 21. |br| See :ref:`Requirements <requirements_supported_platforms>` for more
+  information on CrafterCMS requirements.
+
+.. .. _compatibility-with-3.1.x:
+
+- **Upgrading 3.1.x sites** |br|
+
+  - MongoDB in CrafterCMS Social and Profile has been updated. To upgrade MongoDB in your installation, see
+    :ref:`upgrading-mongodb`
+
+  - CrafterCMS 4.2.x requires Git. |br| See :ref:`Requirements <requirements_supported_platforms>` for more
```

Comment:
Is this different from 4.1? If not, can we say something like CrafterCMS 4.1.x onward ...